product_docs/docs/pgd/3.7/bdr/conflicts.mdx

```diff
@@ -883,12 +883,11 @@ Although the counter is incremented only on UPDATE, this technique allows
 This approach resembles Lamport timestamps and fully prevents
 the ABA problem for conflict detection.
 
-!!! Note
-    The row-level conflict resolution is still handled based on the
-    [Conflict Resolution] configuration even with row versioning. The way
-    the row version is generated is only useful for detection of conflicts
-    and should not be relied to as authoritative information about which
-    version of row is newer.
+!!! Note The row-level conflict resolution is still handled based on the
```

Comment:
The first part is being displayed as part of the title (up to [... based on the]
```suggestion
!!! Note 
    The row-level conflict resolution is still handled based on the
```